mobile/src/reducers/settings.js

```diff
@@ -22,7 +22,7 @@ export const settings = (state = initialState, action) => {
     case INIT_STATE:
       return initialState
     case SET_CLIENT:
-      return { ...state, client: action.client, authorized: true }
+      return { ...state, authorized: true }
```

Comment:
This seems to be a refactoring's debt as `SET_CLIENT` is not accurate anymore for this action's reducer.